### PR TITLE
Add sys software suite

### PR DESCRIPTION
**Pull request by System64fumo**

### CODE REVIEW

**Comment by System64fumo** [comment]
@yavko

**Comment by yavko** [comment]
> @yavko

i completely missed this sowwy

File: README.md [hunk 1]
```diff
@@ -108,10 +110,12 @@ These technically aren't hyprland plugins, but extend hyprland functionality usi
 - [SwayOSD](https://github.com/ErikReider/SwayOSD) ![rust][rs] (GNOME like OSD written in gtk)
 - [Avizo](https://github.com/misterdanb/avizo) ![vala][va] (macOS like OSD written in gtk, also comes with nice scripts)
 - [Wob](https://github.com/francma/wob) ![c][c] (dead simple OSD inspired by xob)
+- [sysvol](https://github.com/System64fumo/sysvol) ![c++][cpp] (A basic GTK4 volume indicator)
```
Comment (yavko) [inline README.md:113]: why don't u stay consistent and have this also say gtkmm4

**Comment by System64fumo** [comment]
All's good!

File: README.md [hunk 2]
```diff
@@ -123,6 +127,7 @@ These technically aren't hyprland plugins, but extend hyprland functionality usi
 - [anyrun](https://github.com/Kirottu/anyrun) ![rust][rs] (A wayland native, highly customizable runner, similar to krunner)
 - [hypr-empty](https://github.com/nate-sys/hypr-empty) ![rust][rs] (Open a runner on empty workspaces)
 - [nwg-drawer](https://github.com/nwg-piotr/nwg-drawer) ![go][go] (Gnome-ish application drawer)
+- [sysmenu](https://github.com/System64fumo/sysmenu) ![c++][cpp] (A simple and efficient gtk4 program launcher)
```
Comment (yavko) [inline README.md:130]: here too

**Review by yavko — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (System64fumo) [reply README.md:113]: Honestly?
Fair point

File: README.md (hunk 2 above)
Comment (System64fumo) [reply README.md:130]: Very wel, ill change it

**Comment by System64fumo** [comment]
Should be good now lmk

**Review by yavko — APPROVED** [review]

**Comment by yavko** [comment]
aight sick

**Comment by System64fumo** [comment]
Cool beans, Just as a heads up, sysvol will be renamed this week, likely in a few commits so uhh yeah keep ur heads up

**Comment by yavko** [comment]
> Cool beans, Just as a heads up, sysvol will be renamed this week, likely in a few commits so uhh yeah keep ur heads up

oki, can u plz pr it then

**Pull request merged**